mne/gui/_viewer.py

```diff
@@ -41,20 +41,27 @@ def _m_fmt(x):
                                    evaluate=float,
                                    format_func=lambda x: '%0.1f' % x)
 
-_BUTTON_WIDTH = -75
-_RAD_WIDTH = -70  # radian floats
+_BUTTON_WIDTH = -70
+_OMIT_BUTTON_WIDTH = -100
+_RAD_WIDTH = -65  # radian floats
 _M_WIDTH = _RAD_WIDTH  # m floats
-_MM_WIDTH = _RAD_WIDTH  # mm floats
-_INC_BUTTON_WIDTH = -30  # inc/dec buttons
+_MM_WIDTH = -50  # mm floats
+_INC_BUTTON_WIDTH = -25  # inc/dec buttons
 _SCALE_WIDTH = -50  # scale floats
 _WEIGHT_WIDTH = -40  # weight floats
 _RAD_STEP_WIDTH = -50
+_VIEW_BUTTON_WIDTH = -50
 _M_STEP_WIDTH = _RAD_STEP_WIDTH
-# width is optimized for macOS and Linux avoid a horizontal scroll-bar;
-# might benefit from platform-specific values
-_COREG_WIDTH = -300
-_TEXT_WIDTH = -240
+# width is optimized for macOS and Linux avoid a horizontal scroll-bar
+# even when a vertical one is present
+_COREG_WIDTH = -280
+_TEXT_WIDTH = -250
+_REDUCED_TEXT_WIDTH = _TEXT_WIDTH - 40 * np.sign(_TEXT_WIDTH)
```

Comment:
Vertical bar still present on my macOS. In addition, I wonder if this change is responsible for a juuuust slightly too tight area for various buttons, such as

![image](https://user-images.githubusercontent.com/5610076/37175705-9f481df6-231a-11e8-82f7-a6c2e51888b0.png)

Very strangely: only the left panel has a scroll bar, the right not. Is there any way of forcefully preventing it (did you on the right)? Those of us with limited real-estate will just have to resize the panels so that we both see the panels and the scene. In fact, both left and right have a minimum width: try to make them more narrow and they fold out of sight altogether; brilliant!

Reply:
> Vertical bar still present on my macOS.

Which vertical bar? You mean the scroll bar? Yes it might be present on left, right, or both, depending on your layout. I'll tweak the button size, though.

Reply:
... there should be no *horizontal* scroll bars on either, though.

Reply:
T/R/F/L button sizes tweaked, see if it works for you now.

Reply:
button sizes ok, but _horizontal_ scroll bar persist (on left, not right?!)

![image](https://user-images.githubusercontent.com/5610076/37176744-92597a92-231d-11e8-93b3-c60963a5004a.png)